plugin/documents.py

```diff
@@ -483,9 +483,22 @@ def _on_code_actions(self, responses: CodeActionsByConfigName) -> None:
             scope = 'markup.changed'
             icon = 'Packages/LSP/icons/lightbulb.png'
         else:  # 'annotation'
+            annotations_template = """
+            <body id="lsp-line-annotation">
```

Comment:
The other stylesheets (popup.css and an sheets.css) use `.lsp_popup` and `.lsp_sheet` respectively. Would like this to follow that convention which means it should be a class like `lsp_code_action`, for example.

Reply:
This PR follows the best practices for minihtml content as described in https://www.sublimetext.com/docs/minihtml.html#best_practices.
For the simple annotations here it is probably not very relevant, but *in theory* this allows color schemes to tweak the CSS, for example to adjust the background color.

Afaik the `.lsp_popup` class is only used if the minihtml content is passed to mdpopups, which circumvents the intended way and forces its styles over color schemes anyway.

The color boxes at https://github.com/sublimelsp/LSP/blob/0a3ad53d56dc8e628ee051eb9a903ccf2e8b4193/plugin/core/views.py#L553-L561 are another example similar to this PR where LSP doesn't use mdpopups and follows the best practices to allow color schemes to tweak styles if desired.

Reply:
OK, so let's at least change to `lsp-code-action` because "annotation" is just an implementation detail.

Reply:
If you apply the fix for code lenses too, as suggested by rwols above, this HTML template can probably be used for both of them. Maybe then `lsp-annotation` would actually be okay?